server/models/view-models/variationDetails.ts

```diff
@@ -0,0 +1,55 @@
+import { getErrorsViewModel, isNullOrUndefined } from '../../utils/utils'
+import { VariationDetails } from '../VariationDetails'
+import { VariationDetailsFormData } from '../form-data/variationDetails'
+import { ValidationResult } from '../Validation'
+
+const deserialiseDate = (dateString: string | null) => {
+  if (dateString === null || dateString === '') {
+    return {
+      day: '',
+      month: '',
+      year: '',
+    }
+  }
+
+  const date = new Date(dateString)
+
+  return {
+    day: date.getDate().toString(),
+    month: (date.getMonth() + 1).toString(),
```

Comment:
why +1?

Reply:
Months are zero indexed

Reply:
Oh, it's one of those using zero indexing thing 🤦